Add license field to @gluestack-ui/* scoped packages

noahlozevski opened the pull request:
The license field wasn't being published in the newer @gluestack-ui/* scoped packages on NPM. Example with [@gluestack-ui/overlay](https://www.npmjs.com/package/@gluestack-ui/overlay), where the license field shows as "none".

I assume the license to be applied to these packages would be the same as the code in the rest of the repo, which is currently set to MIT license. I copied this attribute to all the packages manually, and bumped the version so it can be published and consumed by others upstream.

Adding the correct license is a requirement the CI tool I am working out of right now. I hope if the MIT license isn't the correct license for these packages, I can at least get the discussion going around the topic here.

akash3gtm commented:
Hi @noahlozevski, thank you for this.

could you please 
- take the latest pull from patch branch.
- update only the licensing but not the package version.
- also check in all the packages.
- package version is handled by changeset and we update that when we release the packages i.e. after merging in the patch branch.

noahlozevski commented:
Hey! Thanks for the feedback. I just updated the PR with the license fields for all the package.json's being published in the repo, and I have rebased it on top of main. Let me know if you need anything else to merge it.

noahlozevski commented:
@akash3gtm Any chance you could re-review?

Sanchitv3 commented:
@rovaron-cit ,
Thanks for flagging this out! We’ve just released the latest version of gluestack-ui v3 under the MIT license — you can now use it without any licensing concerns. 🚀
